x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/execute.cy.ts

```diff
@@ -16,12 +17,11 @@ import {
 import type { IndexedFleetEndpointPolicyResponse } from '../../../../../../common/endpoint/data_loaders/index_fleet_endpoint_policy';
 import { createAgentPolicyTask, getEndpointIntegrationVersion } from '../../../tasks/fleet';
 
-import { login } from '../../../tasks/login';
 import { enableAllPolicyProtections } from '../../../tasks/endpoint_policy';
 import { createEndpointHost } from '../../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../../tasks/delete_all_endpoint_data';
 
-describe('Response console', { tags: ['@ess', '@serverless'] }, () => {
+describe('Response console', { tags: ['@ess', '@serverless', '@serverlessQA'] }, () => {
```

Comment:
Have you checked already that this test is stable on an MKI environment?

Reply:
We will make a decision as a team of what tests to promote for release pipeline. Removed for time being.